src/script/check_commands.sh

```diff
@@ -1,5 +1,6 @@
-#!/bin/sh
-git grep COMMAND\( | grep -o "(\"[a-zA-z ]*\"" | grep -o "[a-zA-z ]*" > commands.txt
+#!/usr/bin/env bash
```

Comment:
i think there is no need to change it to bash. as this script is POSIX compliant.